.circleci/workflows.yml

```diff
@@ -278,7 +278,7 @@ jobs:
       FERROCENE_HOST: x86_64-unknown-linux-gnu
       SCRIPT: |
         ferrocene/ci/scripts/llvm_cache.py download
-        ./x.py test --coverage=library library/core --no-doc
+        ./x.py test --coverage=library library/core --no-doc --stage=1
```

Comment:
👍 as discussed internally, we're going to merge this as a short-term fix while I work on a long-term fix.

Reply:
#1978